exercises/nth-prime/tests/nth-prime.rs

```diff
@@ -2,29 +2,29 @@ extern crate nth_prime as np;
 
 #[test]
 fn test_first_prime() {
-    assert_eq!(np::nth(1), Ok(2));
+    assert_eq!(np::nth(1), Some(2));
 }
 
 #[test]
 #[ignore]
 fn test_second_prime() {
-    assert_eq!(np::nth(2), Ok(3));
+    assert_eq!(np::nth(2), Some(3));
 }
 
 #[test]
 #[ignore]
 fn test_sixth_prime() {
-    assert_eq!(np::nth(6), Ok(13));
+    assert_eq!(np::nth(6), Some(13));
 }
 
 #[test]
 #[ignore]
 fn test_big_prime() {
-    assert_eq!(np::nth(10001), Ok(104743));
+    assert_eq!(np::nth(10001), Some(104743));
 }
 
 #[test]
 #[ignore]
 fn test_zeroth_prime() {
-    assert!(np::nth(0).is_err());
+    assert_eq!(np::nth(0), None);
```

Comment:
Changing this from `assert!` to `assert_eq!` was a good call, the error message goes from:
```
---- test_zeroth_prime stdout ----
	thread 'test_zeroth_prime' panicked at 'assertion failed: np::nth(0).is_none()', tests/nth-prime.rs:28:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
to the much clearer
```
---- test_zeroth_prime stdout ----
	thread 'test_zeroth_prime' panicked at 'assertion failed: `(left == right)`
  left: `Some(1)`,
 right: `None`', tests/nth-prime.rs:28:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

You can see the [full output](https://gist.github.com/cbzehner/5a7dede3a62048457aabebefc4bc007a#file-is_none_vs_none-sh) of both options here.